### ISSUE TITLE
Use compass?

### ISSUE THREAD

**Issue by pzi** [studiozimmerli/SASS-Susy-Vs-Neat#1]
What if you used [compass](compass-style.org)?

**Comment by studiozimmerli** [comment]
Do you think using Compass with Susy 2 has an impact for the overall compile time (in a positive way)?

[Compass](http://compass-style.org) is a Framework like [Bourbon.io](http://bourbon.io) (plus some extra stuff). Used it for a while, but not a big fan...

Susy 2 in the meantime don't require Compass anymore, so I dont use it (keep things simple & lightweight). On the other hand, an additional Framework (like Compass) needs additional time to compile.

**Comment by pzi** [comment]
Fair enough... I am not sure what you need Susy2 for. In fact I am not sure what kind of need you are trying to satisfy with any framework? Also it depends what kind of lengths you want to go.

As far as I can tell they all use Ruby Sass, which is slower.

On the weekend I changed [portefeuille](http://github.com/pzi/portefeuille) to use [libsass](https://github.com/sass/libsass) instead of Compass. The reason for the change was my laptop acting up with Ruby versions/gems. So I thought 'fuck it' and installed libsass. All works fine. Just means I don't get to use Compass for that project any more. Which is perfectly fine for this project. All I really needed from it were the mixins for vendor prefixing. However [autoprefixer](https://github.com/postcss/autoprefixer) does a way better job than Compass at this.

I used to have and use Codekit 1.. but then I started my job here and it was all done with Ruby anyway. No need for an extra precompiler. Later on we added Node (Grunt & Gulp) for WordPress themes... :)

Not sure how much you have tied Codekit2 into your workflow, but in general, I'd recommend switching to [Grunt.js](gruntjs.com) or [Gulp.js](http://gulpjs.com/)... make it so much fun :D

**Comment by studiozimmerli** [comment]
Using Susy or Neat as responsive Grid toolset (doing the math, but both not really Frameworks). Because Neat depends on [Bourbon](http://bourbon.io), sometimes I pick useful Mixins from it (vendor prefixing, animations, ...), just for convenience.

You can't compare libsass with Compass (or CodeKit or any Framework). It depends on which compiler you use "under the hood" (the Ruby one or libasass (native C implementation)). IMHO you can use Compass for convenience and compile it with libsass (instead of Ruby SASS).

I also tried libsass a while ago. Its realy fast (because it's a port written in native C) but stucks with new SASS features (like [map](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#maps)). Thats a "No Go" for me.

I use CodeKit mainly for very small (single page) projects or just for prototyping. For normal projects i'm also going with Grunt.js.
